go/engine/login_state_test.go

```diff
@@ -395,43 +395,6 @@ func TestLoginWithPassphraseNoStore(t *testing.T) {
 	}
 }
 
-// Test that the login flow with passphrase and with saving the secret
```

Comment:
after calling `localDb.Nuke()` during logout this test broke. @patrickxb gave thumbs to remove